domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ClaimData.java

```diff
@@ -72,6 +72,9 @@
     @Valid
     private final HousingDisrepair housingDisrepair;
 
+    @Valid
+    private final Timeline timeline;
```

Comment:
shouldn't this also be on the response? I can't see it in this PR

Reply:
it would be in separate PR for that story